config/routes/claims.rb

```diff
@@ -56,6 +56,14 @@
 
     resources :claims, only: %i[index show] do
       get :download_csv, on: :collection
+
+      unless HostingEnvironment.env.production?
```

Comment:
Instead of using this environment check, you should think about using feature flag for each specific feature/route.